### PR TITLE
Update README with information on bots API

### PR DESCRIPTION
**Pull request by autumnfound**

### CODE REVIEW

**Review by mbarbero — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -22,6 +31,10 @@ example) Signed-off-by: Martin Lowe <martin.lowe@eclipse-foundation.org>
 
 While these rules apply to all project repositories, any non-project repositories will also be checked. The messages indicate the failures as warnings to the consuming service (like Gitlab or Gerrit) unless 'strict mode' is enabled for the service. Whether or not a repository is tracked (or if it is a project repository) is determined by its presence as a repository directly linked to an active project within the [PMI](https://projects.eclipse.org/), as reported by the [Projects API](https://api.eclipse.org/#tag/Projects). 
 
+## Submitting applications for new bots
+
+Does your project require a bot to mroe effectively run, or additional bots for new functionality? To submit requests for new bots to be registered within our API, please see the [Project Bots API repository](https://github.com/EclipseFdn/projects-bots-api) and create an issue. We will require the projects name, your Eclipse Foundation username (for verification of authority), the name of the bot, the email address used to register the account for the bot, and a quick description of what the bot will do so that we can properly document it internally.
```
Comment (fredg02) [inline README.md]: mroe -> more

I'm not sure if this section is rather confusing for projects/committers. Usually a project wants to use a service like GitHub, GitLab, OSSRH, etc and requests that in a Bugzilla issue. We then create the corresponding bot users and register them with the bots API. This section gives the impression, that projects should create bot users themselves or get existing bots registered, so the ECA check won't fail. That use-case is quite rare (so far). I can only think of dependabot and the openshift-ci-bots atm.

Comment (autumnfound) [reply README.md]: This documentation was quickly made in response to a request by Chris to direct questions to the right areas of our code base. If its just the wording I'm definitely open to suggestion! I just did a quick write up based on knowledge.

Maybe split this into 2 paragraphs? First would indicate that bugzilla issues should be used for general bot registration, and the second would indicate integrations with existing bots should make a request through the Bots API repo. WDYT?

Comment (fredg02) [reply README.md]: To keep it short and simple, I would only mention that issues with the ECA check related to existing bot accounts (e.g. dependabot) should be opened in the projects-bot-api repo.

**Review by fredg02 — APPROVED** [review]
:+1:

File: README.md (hunk 1 above)
Comment (autumnfound) [reply README.md]: Done

**Pull request merged**